app/src/main/java/kr/ac/konkuk/gdsc/plantory/presentation/plant/AddPlantFragment.kt

```diff
@@ -1,4 +1,102 @@
 package kr.ac.konkuk.gdsc.plantory.presentation.plant
 
-class AddPlantFragment {
+import android.app.DatePickerDialog
+import android.net.Uri
+import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
+import android.view.View
+import android.widget.ArrayAdapter
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.core.widget.addTextChangedListener
+import androidx.databinding.adapters.TextViewBindingAdapter.setText
+import com.bumptech.glide.Glide
+import dagger.hilt.android.AndroidEntryPoint
+import kr.ac.konkuk.gdsc.plantory.R
+import kr.ac.konkuk.gdsc.plantory.databinding.FragmentAddPlantBinding
+import kr.ac.konkuk.gdsc.plantory.util.binding.BindingFragment
+import java.util.Calendar
+
+@AndroidEntryPoint
+class AddPlantFragment : BindingFragment<FragmentAddPlantBinding>(R.layout.fragment_add_plant) {
+
+    private var selectedImageUri: Uri? = null
+    private lateinit var viewModel: AddPlantViewModel
```

Comment:
아직 기능 로직에 들어가지 않아 저처럼 안해둔 거일수도 있지만 혹시나 도움이 될까 미리 알려드립니다 ~! 
저도 이참에 다음 pr에 다 바꿔두어야겠네요 ㅎㅎㅎ
```suggestion
     private val viewModel: AddPlantViewModel by viewModels()
```